### PR TITLE
Fixes #31320 - Remove old button css styling

### PR DESCRIPTION
**Pull request by jturel**
This fixes the the whited-out styling on the user dropdown on the top right of the sync status page. It's a little scary removing a large chunk of css like this, but I'm almost positive it's not affecting other pages and that's what I'd like the reviewer to validate. I was able to confirm, at least, that the subclasses like `actionlink` and `formbutton` aren't used anymore.

### CODE REVIEW

**Review by jeremylenz — COMMENTED** [review]
Thanks for improving our final remaining ERB page in Katello, @jturel :)

Suggestion below.

File: app/assets/stylesheets/katello/katello.scss [hunk 1]
```diff
@@ -163,78 +163,6 @@ input:focus {
   @extend .status_exclamation_icon;
 }
 
-/* BUTTONS */
-input[type='submit'], button, .button {
```
[the remark this answers is not in the document]
Comment (jturel) [reply app/assets/stylesheets/katello/katello.scss:167]: I interpreted the spirit of your feedback and just added `btn btn-default` to it, and I think it looks pretty good (consistent with pf pages). Let me know what you think

**Review by jeremylenz — APPROVED** [review]
@jturel When the button is disabled, it's doing a weird thing where it gets a black border if you click on it (I think it's something with `border` when `:active`).  But a similar behavior is also present in master, so I'll approve and leave it up to you whether you want to spend more time on it.

Other than that, works fine for me :)

**Comment by jturel** [comment]
> @jturel When the button is disabled, it's doing a weird thing where it gets a black border if you click on it (I think it's something with `border` when `:active`). But a similar behavior is also present in master, so I'll approve and leave it up to you whether you want to spend more time on it.
> 
> Other than that, works fine for me :)

ACK. I'm gonna leave the button as-is!

**Pull request merged**